src/ni/vsbuild/packages/AbstractPackage.groovy

```diff
@@ -35,6 +35,17 @@ abstract class AbstractPackage implements Buildable {
       return []
    }
 
+   protected def getQuarterlyDisplayVersion() {
+      // Rather than try to finalize the versioning policy now,
+      // punt and just handle the immediate case.
+      def baseVersion = getBaseVersion()
+      if (baseVersion == "23.0.0") {
+         return "2023 Q1"
+      }
```

Comment:
I'll create a tech debt item tracking that we need to extend this logic after the current release.